Automattic/wp-calypso
# Move GSuite Components Under Email Module

#### Changes proposed in this Pull Request

* Move GSuite Components Under Email Module

#### Testing instructions

1. Navigate to `/domains/manage/:domainName/email/:siteSlug` for a domain without G Suite
1. Confirm that the G Suite Call to Action correctly renders <img width="1059" alt="Screen Shot 2019-03-12 at 9 05 05 AM 1" src="https://user-images.githubusercontent.com/2810519/54215997-1939f780-44a6-11e9-801e-6b741f0254c0.png">
1. Navigate to `/domains/manage/:domainName/email/:siteSlug` for a domain with G Suite
1. Confirm that the G Suite Users list correctly renders <img width="1060" alt="Screen Shot 2019-03-12 at 9 07 55 AM" src="https://user-images.githubusercontent.com/2810519/54216158-59997580-44a6-11e9-9587-286c5ed48744.png">
1. Add 2 users via click the "Add G Suite Users" button ( but do not purchase )
1. Confirm that they were correctly added to the cart
